coretex/coretex/annotation/image/coretex_format.py

```diff
@@ -145,42 +145,42 @@ def extractBinaryMask(self, width: int, height: int) -> np.ndarray:
 
         return binaryMask
 
-    def centroid(self) -> Tuple[float, float]:
+    def centroid(self) -> Tuple[int, int]:
         """
             Calculates centroid of segmentations
 
             Returns
             -------
-            Tuple[float, float] -> x, y coordinates of centroid
+            Tuple[int, int] -> x, y coordinates of centroid
         """
 
         flattenedSegmentations = [element for sublist in self.segmentations for element in sublist]
 
         listCX = [value for index, value in enumerate(flattenedSegmentations) if index % 2 == 0]
-        centerX = fsum(listCX) / len(listCX)
+        centerX = int(sum(listCX) / len(listCX))
```

Comment:
instead of converting float to integer after division use "//" operator for division which returns an integer